tests/go.mod

```diff
@@ -2,6 +2,8 @@ module go.etcd.io/etcd/tests/v3
 
 go 1.22
 
+toolchain go1.22.2
```

Comment:
Thanks for raising this @MadhavJivrajani. Please see my suggestion here https://github.com/etcd-io/etcd/pull/17733#discussion_r1557058353.  Keen to see if we can avoid putting hardcoded go version into more files if possible as we update frequently and it might be easy to miss updating it in this lower version `go.mod`.

Reply:
Done, thanks!